src/init/variables/computers/ArgExtractor.ts

```diff
@@ -0,0 +1,20 @@
+import { ValueComputer } from './ValueComputer';
+
+/**
+ * A simple {@link ValueComputer} that extracts a single value from the input map.
+ * Returns the default value if it was defined in case no value was found in the map.
+ */
+export class ArgExtractor extends ValueComputer {
```

Comment:
I went for `KeyExtractor` (or `Key...` if that part still changes) since the focus is mostly on the fact that it directly copies a specific entry from the incoming values. The fact that you can set a default is a bonus.